samples/bidirectional-chat/README.md

```diff
@@ -0,0 +1,115 @@
+# Azure function bidirectional chatroom sample
+
+This is a chatroom sample that demonstrates bidirectional message pushing between Azure SignalR Service and Azure Function in serverless scenario. It leverage the "upstream" provided by Azure SignalR Service that features proxying messages from client to upstream endpoints in serverless scenario. Azure Functions with signalr trigger binding allow you to write code to receive and push messages in several languages, including JavaScript, Python, C#, etc.
```

Comment:
>upstream [](start = 164, length = 8)

strong "upstream"? do we have any document for detail explanation about upstream that can be link here?

Reply:
We don't have public document about `upstream` now. We can update it when we have it.